internal/c99/headers/windows_amd64/builtin.h

```diff
@@ -0,0 +1,137 @@
+//  Copyright 2017 The CCIR Authors. All rights reserved.
+//  Use of this source code is governed by a BSD-style
+//  license that can be found in the LICENSE file.
+
+// +build ignore
+
+#ifndef _BUILTIN_H_
+#define _BUILTIN_H_
+
+#include "predefined.h"
+
+typedef struct {
+	void *_[2];		// Go *[]interface{}
+} *__builtin_va_list;
+
+#define _MSC_VER 1900
+#define _MSVCRT_
+#define _MSC_EXTENSIONS
+#define __int8 char
+#define BYTE unsigned char
+#define __int16 short
+#define __int32 int
+#define __int64 long long
+// disable SSE and other instrisics we dont support
+#define _X86INTRIN_H_INCLUDED 1
+#undef __SSE__
+#undef __SSE2__
+#undef __MMX__
+// bypass bug that fails for: #define MICROSOFT_WINDOWS_WINBASE_H_DEFINE_INTERLOCKED_CPLUSPLUS_OVERLOADS (_WIN32_WINNT >= 0x0502 || !defined (_WINBASE_))
+#define MICROSOFT_WINDOWS_WINBASE_H_DEFINE_INTERLOCKED_CPLUSPLUS_OVERLOADS 1
+#define __unaligned
+#define __inline__ inline
+#define __forceinline inline __attribute__((__always_inline__,__gnu_inline__))
+
+// TODO: __builtin_fprintf(stderr, "%s:%i.%s STUB %s called!\n", __FILE__, __LINE__, __func__, #x), __builtin_abort(), 0)
+#define abort_stubbed(x) 0
+
+// maybe: `#define __CRT__NO_INLINE` to remove some unnecessary stuff
+// Implementing __readgsqword as macro also prevents an anonymous union access 
+// (which isn't supported in C99 actually WTF)
```

Comment:
I don't mind the `WTF`, but please consider some other form for others out there ;-)